README.md

```diff
@@ -1,3 +1,14 @@
 # TartanAir
 
-Hello and welcome to the official TartanAir repository. This repository includes a set of tools that complement the [TartanAir Dataset](https://www.tartanair.org/)). 
+Hello and welcome to the official TartanAir repository. This repository includes a set of tools that complement the [TartanAir Dataset](https://www.tartanair.org/)).
+
+## Installation Instructions
```

Comment:
Thanks for starting this PR, Akash!

Could you please change this to distinguish between `pip` installation and local installation? 

```suggestion
## Installation Instructions
`pip install tartanair`

## Local Installation Instructions
```